plugins/nanospell/plugin.js

```diff
@@ -347,6 +347,13 @@
 			function stop() {
 				editor.getCommand('nanospell').setState(CKEDITOR.TRISTATE_OFF);
 				commandIsActive = false;
+
+				// clear any currently queued spellcheck
+
+				if (self._timer) {
```

Comment:
This is for the singular timer that is used to prevent stacking spellchecks.

I agree now the name is bad, but this is likely to change as a result of @sbaughman 's work anyway and didn't want this simple PR for Travis to cause a bunch of surprise merge conflicts.

The stop() is never actually used in real life right now, since we do not put the button to start/stop spellcheck in the toolbar.